catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -347,12 +347,18 @@ public void init() {
                     .map(Result::getMetacard)
                     .map(metacard -> {
                         boolean isSubscribed = ids.contains(metacard.getId());
-
-                        return ImmutableMap.builder()
-                                .putAll(transformer.transform(metacard))
-                                .put("subscribed", isSubscribed)
-                                .build();
+                        try {
+                            return ImmutableMap.builder()
+                                    .putAll(transformer.transform(metacard))
+                                    .put("subscribed", isSubscribed)
+                                    .build();
+                        } catch (RuntimeException e) {
+                            LOGGER.warn(
+                                    "Could not transform metacard. WARNING: This indicates there is invalid data in the system. Metacard title [{}]", metacard.getTitle());
```

Comment:
oh. I just dropped my second commit, didn't realize it I dropped those out too. will fix